scripts/extract_dts_includes.py

```diff
@@ -404,6 +404,20 @@ def extract_single(node_address, yaml, prop, key, prefix, defs, def_label):
 
   return
 
+def extract_string_prop(yaml, node_address, key, label, defs):
+
+  prop_def = {}
+  node = reduced[node_address]
+  prop = node['props'][key]
+
+  k = convert_string_to_label(key).upper()
+  prop_def[label] = "\"" + prop + "\""
+
+  if node_address in defs:
+    defs[node_address].update(prop_def)
+  else:
+    defs[node_address] = prop_def
+
```

Comment:
I wonder if the extract_property shouldnt be more generic and use the type to decide what to do about the value.  As it is right now, the extract_single is being called within the extract_property.  If extract single handled the stringification, we'd be ok to just use that.  The only other issue to deal with is the label.